project_share/templates/project_share/user_detail.html

```diff
@@ -11,6 +12,24 @@
         </h1>
     </div>
 
+    <h3>Update Profile:</h3>
```

Comment:
Please show avatar image... we may have to have some level of censorship for this at some point...